### ISSUE TITLE
The `loopEnabled` configuration doesn't loop a video.

### ISSUE THREAD

**Issue by EvgenyKarkan** [SvenTiigi/YouTubePlayerKit#68]
### What happened?

I might be missing something, but it appears evident that enabling the `loopEnabled` feature should make a video repeat continuously. However, for some reason, the video doesn't repeat and just ends.

### What are the steps to reproduce?

``` Swift
func setupYouTubeView() {
    let configuration = YouTubePlayer.Configuration(
        automaticallyAdjustsContentInsets: true,
        fullscreenMode: .system,
        showControls: true,
        loopEnabled: true,
        useModestBranding: true
    )

    youTubePlayer = YouTubePlayer(
        source: .url("https://youtube.com/watch?v=YE7VzlLtp-4"),
        configuration: configuration
    )

    youTubeView = YouTubePlayerHostingView(player: youTubePlayer!)
    youTubeView?.autoresizingMask = [.flexibleHeight, .flexibleWidth]

    containerView.addSubview(youTubeView!)
}
```

### What is the expected behavior?

I had the expectation that the video would repeat in a continuous loop.

**Comment by SvenTiigi** [comment]
Hi @EvgenyKarkan,

Please check out the [develop](https://github.com/SvenTiigi/YouTubePlayerKit/tree/develop) branch which contains a fix for the loop parameter.

**Comment by EvgenyKarkan** [comment]
Thank you very much, I trust you the issue is fixed.
Please feel free to close the current ticket as you see fit. 
I am not blocked at all and can wait until the new release is available, so I can pull the updated SPM package.

**Comment by SvenTiigi** [comment]
Thanks! 
I will let you know when a new release is available ✌️

**Comment by SvenTiigi** [comment]
The fix is now available with the latest version [`1.5.1`](https://github.com/SvenTiigi/YouTubePlayerKit/releases/tag/1.5.1)
